src/map/utils/charutils.cpp

```diff
@@ -1683,7 +1683,7 @@ namespace charutils
         {
             auto removeSlotID = PItem->getRemoveSlotId();
 
-            for (auto i = 0; i < sizeof(removeSlotID) * 8; ++i)
+            for (auto i = 0; i < SLOT_FEET + 1; ++i)
```

Comment:
probably make that "i <= SLOT_FEET", makes a bit more sense than SLOT_FEET + 1

on the other hand, maybe don't change it at all, in case something ever removes belts or backs or something?